example_cases/SPARC_PRD/input.yaml

```diff
@@ -39,8 +41,7 @@ algorithms:
   # a scaling for lambda_q to calculate a corresponding q_parallel, and
   # then use the two-point-model to determine how much edge seeding we
   # require to protect the divertor.
-  - calc_P_SOL
-  - calc_average_total_pressure
+  # - calc_P_SOL
```

Comment:
I think we should not modify the original SPARC_PRD example in this PR. Can you instead make a new input_constantPSOL.yaml with the changes that demonstrates the effect of your code?

Reply:
Should I keep the calc_mcrf_from_fixed_P_sol algorithm in the yaml file too?